content/en/service_management/service_level_objectives/_index.md

```diff
@@ -236,7 +249,7 @@ When you apply a correction, the time period you specify is dropped from the SLO
 - For monitor-based SLOs, the correction time window is not counted.
 - For metric-based SLOs, all good and bad events in the correction window are not counted.
```

Comment:
Add a line for time slice

- For Time Slice SLOs, the correction time window is treated as uptime.

Reply:
```suggestion
- For metric-based SLOs, all good and bad events in the correction window are not counted.
- For Time Slice SLOs, the correction time window is treated as uptime.
```